mne/io/nirx/tests/test_nirx.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Authors: Robert Luke  <mail@robertluke.net>
+#          simplified BSD-3 license
+
+import os.path as op
+
+from mne.io import read_raw_nirx
+from mne.utils import run_tests_if_main
+from mne.datasets.testing import data_path, requires_testing_data
+from ....utils import logger
+
+
+@requires_testing_data
+def test_nirx():
+    """Test reading NIRX files."""
+    fname = op.join(data_path(), 'nirx', 'test_nirx.nxe')
+    fname = '/home/rluke/Documents/Repositories/mne-python/mne/io' + \
+            '/nirx/tests/nirx_15_2_recording'
```

Comment:
Upload to and follow

https://github.com/mne-tools/mne-testing-data#add-or-change-files-in-the-repo-for-use-with-mne

Reply:
Please see https://github.com/mne-tools/mne-testing-data/pull/50

Reply:
Nope, make that https://github.com/mne-tools/mne-testing-data/pull/51